File: js/euler25.js
function euler25(){
    var n1 = 0n, n2 = 1n, count = 1;
    while (String(n2).length < 1000){
        let temp = n1;
        n1 = n2, n2 += temp;
        count++;
    }
    return count;
}

console.log(euler25());

File: go/euler25.go
package main
import (
    "fmt"
    "math/big"
)

func main() {
    n1, n2 := big.NewInt(1), big.NewInt(1)
    sum, temp := new(big.Int), new(big.Int)
    n := 2
    for len(n2.String()) < 1000 {
        temp.Set(n1)
        n1.Set(n2)
        sum.Add(n2, temp)
        n2.Set(sum)
        n++
    }
    fmt.Println(n)
}
